ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/Wizard/ScanConfigWizardForm.tsx

```diff
@@ -178,144 +178,113 @@ function ScanConfigWizardForm({ initialFormValues }: ScanConfigWizardFormProps):
-                                                onClick={onSave}
-                                                isLoading={isCreating}
-                                            >
-                                                Save
-                                            </Button>
-                                        )}
-                                        <Button
-                                            variant="secondary"
-                                            onClick={onBack}
-                                            isDisabled={activeStep.id === firstStepId}
-                                        >
-                                            Back
-                                        </Button>
-                                        <Button variant="link" onClick={openModal}>
-                                            Cancel
-                                        </Button>
-                                        <Modal
-                                            variant="small"
-                                            title="Confirm cancel"
-                                            isOpen={isModalOpen}
-                                            onClose={closeModal}
-                                            actions={[
-                                                <Button
-                                                    key="confirm"
-                                                    variant="primary"
-                                                    onClick={onClose}
-                                                >
-                                                    Confirm
-                                                </Button>,
-                                                <Button
-                                                    key="cancel"
-                                                    variant="secondary"
-                                                    onClick={closeModal}
-                                                >
-                                                    Cancel
-                                                </Button>,
-                                            ]}
-                                        >
-                                            <p>
-                                                Are you sure you want to cancel? Any unsaved changes
-                                                will be lost. You will be taken back to the list of
-                                                scan configurations.
-                                            </p>
-                                        </Modal>
-                                    </>
-                                )}
-                            </WizardContextConsumer>
-                        </WizardFooter>
-                    }
-                />
+                    onSave={onSave}
```

Comment:
Commenting here about previous line, because of changes for consistency in Vulnerability Reporting wizard.

Tagline for landing pages of containers that have wizards:
* Configure scan schedules…
* Configure reports…
* Configure security policies…

Therefore, what do you think about pattern like:
* `navAriaLabel="Scan schedule configuration steps"`
* `navAriaLabel="Vulnerability report configuration steps"`
* `navAriaLabel="Security policy configuration steps"`

Reply:
I prefer what you have above to the existing `navAriaLabel` - it definitely reads more clearly.